### DISCUSSION TITLE
Issue with an XRD texture of an Al alloy rolled sheet

### DISCUSSION THREAD

**Discussion by MartinL75** [mtex-toolbox/mtex#2113] [Ask Anything] [answered]
Hi Ralph and everybody.
Connected with #2111  issue, I found a similar problem regarding the ODF reconstruction from raw pole figures measured by a Philips X'Pert diffractometer. However, in this case I could not solve it by normalizing the corrected PFs, as we can see below.
Here are the raw PFs:
![Al_6061T4-AR_rawPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/74ffc5ec-a8a8-41b0-aade-c3a308b1dae6)

Now the corrected ones (same method as mentioned in #2111):
![Al_6061T4-AR_corrPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/b0d94e51-257b-4878-a4d5-af68f82d976e)

Then normalizing:
![Al_6061T4-AR_corr normPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/bbfe2693-348b-4149-a58f-dde6866dad41)

And the PFs reconstructed from the ODFs are these:
![Al_6061T4-AR_norm reconstPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/16d3b62f-3e80-45f1-a507-2e69bf50eef6)

It appears that the central high intensity of the (111) and (220) poles are unreal.
Moreover, if we calculate the ODF just from the corrected PFs (not normalized), we have:
![Al_6061T4-AR_reconstPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/865086fd-853a-48a2-a18b-de8ffd037a6c)

which also seems strange.

I'm attaching the rw1 files and the script in a zip file. MTEX version is 5.1.1 running with Matlab R20117b.
[Al_6061_T4.zip](https://github.com/mtex-toolbox/mtex/files/14910354/Al_6061_T4.zip)

Thanks a lot in advance for helping.
Regards,

Martín Leonard
IFIR-CONICET/UNR
Rosario, Argentina.

**Comment by ralfHielscher** [comment, marked as answer]
Hi Martin,

sorry for the delay. I was able to produce this one 
![pf](https://github.com/mtex-toolbox/mtex/assets/8619241/0b5f366f-50d6-475a-a7ab-0b3d9b86b020)

the main thing I changed, was to remove the outer rings of the XRD measurements as they are too heavily effect by defocusing, i.e.
```matlab
pfc(pfc.r.theta >= 70*degree) = [];
```

I uploaded this example to https://mtex-toolbox.github.io/ExAlODFReconstruction.html

PS: I made some more minor changes to the ODF reconstructor and PoleFigure object. Therefore, you will not be able to run the code unless you checkout the current developer branch of MTEX from Github. Nevertheless I would be grateful, if you could add some explanations to the site about the material, the measurement process, etc. You can edit this file here https://github.com/mtex-toolbox/examples/blob/master/ExODFReconstruction/ExAlODFReconstruction.m

All then best,
Ralf.

**Reply by MartinL75** [reply]
Hi Ralph, thank you very much!
The final PFs look alright, similar to other experimental textures from rolled and recrystallized aluminum sheets (i.e., dominant 'cube' orientation as in: https://doi.org/10.1016/j.msea.2019.04.091).
I will add ASAP some experimental details to the example file. I will also try to add the steel example previously treated on issue #2111, maybe for next week.
When calculating the ODF using the removed PFs in my MTEX v.5.1.1, I get a slightly different result:
![Al_6061T4-AR_slicedPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/a8c43d70-6ed3-4d58-9737-d713d474d153)
But I think this is due to the different versions used.
Thanks again and regards.

**Reply by MartinL75** [reply]
Hi again, I tried to access the example file to edit it, but I got a 404 error.

**Reply by ralfHielscher** [reply]
Sorry, the link should be working right now. I forgot to push.
Ralf.

**Reply by MartinL75** [reply]
I've just added some comments about experimental data. Please let me know if it works OK (it's my first contribution) and also if you think that I should add some more details.
Regards

**Reply by ralfHielscher** [reply]
Hi Martin,
I can not see you edits. You should proceed as described [here](https://mtex-toolbox.github.io/Documentation) and generate a pull request. You may also send me the changes by mail.
Thank you a lot.
Ralf

**Reply by MartinL75** [reply]
I think the pull request was succesfully generated now. Thanks

**Reply by ralfHielscher** [reply]
Yes I merged it. I will take a while to be visible on the example page.
Thanks a lot.
Ralf.

**Comment by FotisTsiolis** [comment]
@ralfHielscher 
If we have PF data from a powder sample with almost random texture, do we calculate the background and defocusing correction PFs in the same way shown in this example?
